tests/test_diffraction_fft.py

```diff
@@ -258,32 +289,56 @@ def test_fft_diffraction_is_similar_to_raytracing_for_0_exptime():
     [c_x, c_y] = center_of_brighness(brightness)
     # 2 Pixel tolerance:
     np.testing.assert_allclose(
-        np.array([c_x, c_y]), np.array([1923.0, 3161.0]), atol=2.0, rtol=0.0
+        np.array([c_x, c_y]), raytrace_data["c"], atol=2.0, rtol=0.0
     )
+
     angle, angle_stddev = folded_spike_angle(brightness, c_x, c_y, r_min=10.0)
     expected_angle = (
         45.0 * galsim.degrees - config["stamp"]["diffraction_psf"]["rottelpos"]
-    ).deg
+    )
+
     # 1° tolerance:
-    np.testing.assert_allclose(np.rad2deg(angle), expected_angle, atol=1.0, rtol=0.0)
+    np.testing.assert_allclose(
+        np.rad2deg(angle), expected_angle.deg, atol=1.0, rtol=0.0
+    )
+    np.testing.assert_allclose(
+        np.rad2deg(raytrace_data["angle"]), expected_angle.deg, atol=1.0, rtol=0.0
+    )
     expected_max_spike_width = 5.0
     np.testing.assert_array_less(np.rad2deg(angle_stddev), expected_max_spike_width)
     slope, intercept, slope_stderr, intercept_stderr = brightness_log_profile(
         brightness, c_x, c_y
     )
+    # Brightness should decay as ~1/r**2:
     np.testing.assert_allclose(slope, -2.0, atol=0.6, rtol=0.0)
-    np.testing.assert_allclose(intercept, 12.5, atol=3.0, rtol=0.0)
-    # Assert some rough evidence, that brightness decays as
-    # brightness ~ exp(intercept) * r**slope:
+    np.testing.assert_allclose(raytrace_data["slope"], -2.0, atol=0.6, rtol=0.0)
+
+    np.testing.assert_allclose(
+        intercept, raytrace_data["intercept"], atol=3.0, rtol=0.0
```

Comment:
atol=3 still seems high to me.  This is only accurate to about 25% tolerance?

Reply:
They are actually much closer. I set the tolerance to 0.5 now